app/components/publications/data_table_component.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Publications
+  class DataTableComponent < ViewComponent::Base
+    attr_reader :caption, :title, :data
+
+    def initialize(caption:, statistics:)
+      @caption = caption
+      @title = statistics[:title]
+      @data = statistics[:data]
+    end
```

Comment:
Do you think we should decouple the component from this data structure, and pass it in the view instead. That would make it more reusable?

So here it would be:

 def initialize(caption:, statistics:)
      @caption = caption
      @title = title
      @data = data
    end

WDYT?

Reply:
:+1: